readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://DianaFakhim.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://DianaFakhim.github.io/layout_search-bar-airbnb/report/html_report/)
 
```

Comment:
add this to the PR description
If you have question - write into slack chat - fe_chat1